src/pages/home/profile/Profile.tsx

```diff
@@ -161,127 +185,204 @@ const Profile: FunctionComponent = () => {
-        }) => {
-          const showEndAdornment = !!(
-            currentPassword ||
-            newPassword ||
-            confirmPassword
-          );
-
+        { ({ dirty, handleReset }) => {
           return (
             <Form>
-              <Stack
-                sx={ {
-                  display: "grid",
-                  gridTemplateColumns: [
-                    "repeat(1, 1fr)",
-                    null,
-                    "repeat(2, 1fr)",
-                  ],
-                  columnGap: [undefined, undefined, "20px"],
-                  maxWidth: [undefined, undefined, "700px"],
-                  rowGap: ["16px", null, "12px"],
-                } }
-              >
-                <TextInputField
-                  label="FIRST NAME"
-                  name="firstName"
-                  placeholder="First name"
-                  type="text"
-                />
-                <TextInputField
-                  label="LAST NAME"
-                  name="lastName"
-                  placeholder="Last name"
-                  type="text"
-                />
-                <TextInputField
-                  label="EMAIL"
-                  name="email"
-                  placeholder="Email"
-                  type="email"
-                />
-                <TextInputField
-                  label="STAGE NAME"
-                  name="nickname"
-                  placeholder="Stage name"
-                  type="text"
-                />
-                <DropdownSelectField
-                  label="MAIN ROLE"
-                  name="role"
-                  options={ roles }
-                  placeholder="Main role"
-                />
-                <DropdownSelectField
-                  label="MUSIC GENRE"
-                  name="genre"
-                  options={ genres }
-                  placeholder="Music genre"
-                />
-              </Stack>
-              <Stack
-                sx={ {
-                  marginY: 5,
-                  marginX: ["auto", "auto", "unset"],
-                  maxWidth: ["340px", "340px", "700px"],
-                } }
-              >
-                <HorizontalLine />
-              </Stack>
-              <Typography variant="subtitle1" fontWeight={ 700 }>
-                CHANGE PASSWORD
-              </Typography>
-              <Stack
-                sx={ {
-                  marginTop: 2.5,
-                  marginBottom: 7.5,
-                  display: "grid",
-                  gridTemplateColumns: [
-                    "repeat(1, 1fr)",
-                    null,
-                    "repeat(2, 1fr)",
-                  ],
-                  rowGap: ["16px", null, "12px"],
-                  columnGap: [undefined, undefined, "20px"],
-                  maxWidth: [undefined, undefined, "700px"],
-                } }
+              <Box
+                sx={ { maxWidth: ["340px", "340px", "700px"], margin: "0 auto" } }
```

Comment:
The two text input rows start bunching up at the third breakpoint so I'm moving the breakpoint on the Settings page to make it look fluid.

```suggestion
                sx={ { maxWidth: { xs: "340px", lg: "700px" }, margin: "0 auto" } }
```

___
And same change needed for the flex directions below to keep it lined up.

`flexDirection: { xs: "column", lg: "row" },`

Reply:
I think this needs to be its own ticket where we do a find and replace all an test it everywhere. Otherwise, this PR will be bigger than it needs to. What do you think?

Reply:
Agree, as discussed before the artist-portal needs adjustments for mobile responsiveness, probably with some added figma designs, this could be a part of that epic or just a general UI clean up task. All for the separate ticket.